core/tooltip.js

```diff
@@ -32,102 +33,104 @@ goog.require('Blockly.utils.string');
  * eventually unwinds to a string.
  * @typedef {string|{tooltip}|function(): (string|!Function)}
  */
-Blockly.Tooltip.TipInfo;
+let TipInfo;
+exports.TipInfo = TipInfo;
 
 /**
  * Is a tooltip currently showing?
  */
-Blockly.Tooltip.visible = false;
+let visible = false;
+exports.visible = visible;
 
 /**
  * Is someone else blocking the tooltip from being shown?
  * @type {boolean}
- * @private
  */
-Blockly.Tooltip.blocked_ = false;
+let blocked = false;
 
 /**
  * Maximum width (in characters) of a tooltip.
  */
-Blockly.Tooltip.LIMIT = 50;
+const LIMIT = 50;
+exports.LIMIT = LIMIT;
```

Comment:
Are we expecting that developers will want to modify this value?

Reply:
Checked with Beka and she wasn't aware of and couldn't find anyone that was. I take the all-caps to imply that it was intended as a constant as well.

Reply:
I guess the question then is: does it need to be exported?  I don't see reference to it anywhere outside this file (but maybe it's referenced outside this repo?)

Reply:
Looks like it's used in blockly-samples: https://github.com/google/blockly-samples/blob/1be2b748414ef147c8f7d657fa440442c0ef49a8/plugins/block-extension-tooltip/src/tooltip_monkey_patch.ts

Reply:
That's kind of annoying. First party plugins aren't supposed to monkey patch according to [our own criteria](https://developers.google.com/blockly/guides/plugins/overview#first-party-criteria). Sucks that this has to be made public because of that :/